src/sentry/receivers/core.py

```diff
@@ -69,7 +69,9 @@ def create_default_project(id, name, slug, verbosity=2, **kwargs):
     except IndexError:
         user = None
 
-    org, _ = Organization.objects.get_or_create(slug="sentry", defaults={"name": "Sentry"})
+    org, _ = Organization.objects.get_or_create(
+        slug="sentry", defaults={"name": "Sentry"}, id=getattr(settings, "SENTRY_ORGANIZATION", 1)
```

Comment:
🤔 

so this is defaulting the org id to 1? rather than enabling the snowflake id?

Reply:
Should we instead look for a way to enable snowflake id's internally instead?

Reply:
Yes. That's the idea.

Reply:
@nhsiehgit it's already turned on by default here https://github.com/getsentry/sentry/blob/c31d98536a04f8eafd2952d568386321b7aa7b35/src/sentry/conf/server.py#L2824